### PR TITLE
chore: update ti.cloak to include linux binaries

### PR DESCRIPTION
**Pull request by garymathews**
- Update `ti.cloak` to include Linux binaries

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27514)

### CODE REVIEW

**Comment by lokeshchdhry** [comment]
FR Passed.

Studio Ver:       5.1.4.201909061933
SDK Ver:          8.3.0 local build
Xcode Ver:        Xcode 11.1
Appc NPM:         4.2.15
Appc CLI:         7.1.2-master.3
Daemon Ver:       1.1.3
Ti CLI Ver:       5.2.2
Alloy Ver:        1.14.1
Node Ver:         12.13.0
NPM Ver:          6.12.0
Java Ver:         10.0.2
Android Devices:  ⇨ google Pixel (Android 9)

File: android/titanium/src/java/org/appcelerator/titanium/io/TiResourceFile.java [hunk 1]
```diff
@@ -193,6 +192,19 @@ public long size()
 		InputStream is = null;
 		try {
 			is = getInputStream();
+
+			// CipherInputStream.available() always returns 0
+			// Iterate through stream to obtain true size.
+			if (is instanceof CipherInputStream) {
+				long size = 0;
+				long read = 0;
```
Comment (sgtcoolguy) [inline android/titanium/src/java/org/appcelerator/titanium/io/TiResourceFile.java]: technically this should be an int, as #read() returns an int.

**Review by sgtcoolguy — APPROVED** [review]

**Comment by jquick-axway** [comment]
@garymathews, did you remove the `require('Buffer')` (with a capital 'B') from the script that was causing module loading issues on case-sensitive filesystems?

**Comment by garymathews** [comment]
@jquick-axway Yep

File: android/titanium/src/java/org/appcelerator/titanium/io/TiResourceFile.java (hunk 1 above)
Comment (garymathews) [reply android/titanium/src/java/org/appcelerator/titanium/io/TiResourceFile.java]: Ah, forgot to amend this after switching from `.skip()`

**Pull request merged**